libs/execution_context/ebpf_native.c

```diff
@@ -1285,6 +1293,13 @@ _ebpf_native_close_handles_work_item(_In_opt_ const void* context)
         }
     }
 
+    // Detach process from this worker thread.
+    ebpf_platform_detach_process(handle_info->process_state);
+
+    // Release the reference on the process object.
+    ebpf_platform_dereference_process(handle_info->process_handle);
```

Comment:
Is this needed given line 1319 ?

Reply:
Yes, 1319 line is for the cleanup path which is not called in this path.